dbt_sugar/core/task/doc.py

```diff
@@ -56,10 +58,123 @@ def run(self) -> int:
                 account=dbt_credentials.get("account", ""),
                 database=dbt_credentials.get("database", "dwh"),
             ).get_columns_from_table(model, schema)
-
-        self.process_model(model, columns_sql)
+        if columns_sql:
+            self.orchestrator_document_model(model, columns_sql)
         return 0
 
+    def change_model_description(self, content: Dict[str, Any], model_name: str) -> Dict[str, Any]:
+        """Updates the model description from a model.
+
+        Args:
+            content (Dict[str, Any]): content of the schema.yml.
+            model_name (str): model name to change the description from.
+            description (str): description of the model.
+
+        Returns:
+            Dict[str, Any]: with the schema.yml updated.
+        """
+        model_doc_payload: List[Mapping[str, Any]] = [
+            {
+                "type": "confirm",
+                "name": "wants_to_document_model",
+                "message": "Model Description: This is my previously documented model. Document?",
```

Comment:
This will need to be string completed. we should f-string the model description in there.